src/components/pages/dashboard/dashboard.js

```diff
@@ -142,7 +143,13 @@ class DashboardPage extends Component {
         <ContextFilters>
           <div className="timerange-selection dashboard" onClick={this.props.actions.hideFlyout}>
             <div className="last-refreshed-text"> {`${lang.LAST_REFRESHED} | `} </div>
-            <div className="last-refreshed-time">{this.state.lastRefreshed.toLocaleString()}</div>
+            {
+              moment.locale() === 'en'
+              ? <div className="last-refreshed-time">
+                  {moment(this.state.lastRefreshed).format("h:mm:ss A, M/D/YY")}
+                </div>
+              : <div className="last-refreshed-time">{this.state.lastRefreshed.toLocaleString()}</div>
+            }
```

Comment:
Looks like lot of this code is duplicated to get datetime format in approved format. Is it possible add like a helper function where this code can go and reference it at places where we need. We might change the formatting in coming months again and it might be easier to change that.